agents2/intelligent_cropper.py

```
@@ -0,0 +1,345 @@
+        except Exception:
+            return None
+    
+    def _parse_aspect_ratio(self, ratio_str: str) -> float:
+        """Parse aspect ratio string to float"""
+        try:
+            if ':' in ratio_str:
+                width, height = ratio_str.split(':')
+                return float(width) / float(height)
+            else:
+                return float(ratio_str)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)